collections/_posts/2023-11-09-Elongated-structures.md

```diff
@@ -0,0 +1,355 @@
+
+**Curve parameterization**
+Approximate spline to be less sensitive to label or sampling noise. And use the arc-length parametrization method because there is not assumption on the curve type and potential physical constraints.
+
+**Local distance estimation**
+Create a local hull by constructing the smallest enclosing box for each segment.
+Within each bounding box, the point-to-line distance is calculated for every contained spatial location x. The bounding boxes will overlap for non-zero curvatures $$k>0$$ or if the curve is self-intersecting &rarr; multiple distance estimates.
+
+To map $$\underset{x}{f}$$ to a single distance scalar that can be evaluated with $$h(·)$$, there is two options for a reduction function $$h^∗(\underset{x}{f})$$ :
+- Minimum distance : only the estimator with the smallest point-to-line distance is used for evaluating the distance-dependent function. Easily parallelized, it allows the heatmap generation algorithm to be executed at linear time complexity
+- Inverse distance weighting (IDW) : allows the contribution of all distance estimates.
+
+**Selection of the distance-dependant function**
+- frequently observed signal/noise shapes (Gaussian, Laplace, cosine)
+- versatile in their encoding characteristics (triangular, rectangular)
+- custom signal shape in the form of a simulated hollow-core catheter profile
+
+**Extension to 3D**
+The local hull is approximated by extending the two-dimensional bounding boxes to right rectangular prisms.
+
+## Signal extraction and evaluation metrics
+
+Two post-processing protocols for signal recovery and evaluation, with specific curve type and dataset : 
+
+### Center-line extraction and ASSD comparison
+
+Signal estimation as a set of point + post-processing the estimation : 
+- turn it to a binary image (using Otsu’s method, automatic image threshpolding)
+- transform into a curve with uniform width using medial axis transformation
+
+ASSD between two curves $$C$$ and $$C'$$, 
+- minimum euclidiean distance of point $$x$$ in curve $$C$$ to point $$x'$$ on curve $$C'$$ :
+$$d(x, C') = \underset{x′ \in C′}{min} \vert \vert x − x' \vert \vert \underset{2}{}$$ 
+- evaluating this distance for every point $$x \in C$$ and $$x' \in C'$$ and averaging the resulting distance values :  
+
+$$ASSD = \frac{1}{|C| + |C'|}*(\underset{x \in C}{\Sigma}d(x, C') +\underset{x' \in C'}{\Sigma}d(x, C')d(x', C) )$$ 
+
+Advantage : invariant to the width and distribution of the simulated signal and estimated heatmap
+
+Limitations :
+- noisy and self-intersecting heatmap estimates, the thinning can cause branching artifacts 
+- larger signal or heatmap width, longer subsidiary branch
+- effect is negligible as most errors are caused by signal fragmentation in ambiguous regions
+
+### Non-maximum suppression of the heatmap, morphological thinning and F-measure
+
+Interpret the heatmap as a noisy curve probability map :
```

Comment:
A question of interest for me: how do they transform the distance (\in R) to probability (\in [0, 1] ?